PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp

```diff
@@ -777,6 +777,7 @@ INT_PTR CALLBACK FunctionListPanel::run_dlgProc(UINT message, WPARAM wParam, LPA
 
 			_treeViewSearchResult.init(_hInst, _hSelf, IDC_LIST_FUNCLIST_AUX);
 			_treeView.init(_hInst, _hSelf, IDC_LIST_FUNCLIST);
+			_treeView.makeLabelEditable(false);

```

Comment:
Oops... Picked up this change when I pasted my whole file into the editor instead of trying to manually edit within GitHub. I must have overlooked it before when manually editing. Clearly, pasting is the way to go.